docs/plugin-development-guide/summary.rst

```diff
@@ -26,7 +26,7 @@ and even more. The limit is only your imagination (and business value, of course
 our :doc:`customizing guide</customization/index>`.
 
 At the end, do not hesitate to contact us at contact@sylius.com when you manage to implement a new plugin.
-We would be happy to check it out and add it to our `official plugins list`_!
+We would be happy to check it out and add it to our `approved plugins list`_!
```

Comment:
According to https://github.com/Sylius/Sylius/pull/10884/files#diff-a3cc58ce604fdb983d0d20d09c41893cR63 I think it could be interpreted as a list of approved plugins only.
WDYT about:
```suggestion
We would be happy to check it out and add it to our `plugins list`_!
```
?